### ISSUE TITLE
[arm] Test failed: JIT_jit64._opt_cse_HugeField1_HugeField1_._opt_cse_HugeField1_HugeField1_sh

### ISSUE THREAD

**Issue by VincentBu** [dotnet/coreclr#22314]
**Job:**
https://ci.dot.net/job/dotnet_coreclr/job/master/view/arm/job/jitstress/job/arm_cross_checked_ubuntu_jitstress1_tst/50/

**Failed tests:**
JIT_jit64._opt_cse_HugeField1_HugeField1_._opt_cse_HugeField1_HugeField1_sh
JIT_jit64._opt_cse_HugeField2_HugeField2_._opt_cse_HugeField2_HugeField2_sh

**Log:**
```
JIT_jit64._opt_cse_HugeField1_HugeField1_._opt_cse_HugeField1_HugeField1_sh [FAIL]
        
        Assert failure(PID 10693 [0x000029c5], Thread: 10693 [0x29c5]): SUCCEEDED(res)
            File: /mnt/j/workspace/dotnet_coreclr/master/arm_cross_checked_ubuntu/src/unwinder/arm/unwinder_arm.cpp Line: 1540
            Image: /ssd/j/workspace/dotnet_coreclr/master/jitstress/arm_cross_checked_ubuntu_jitstress2_jitstressregs0x80_tst/bin/tests/Linux.arm.Checked/Tests/Core_Root/corerun
        
        /ssd/j/workspace/dotnet_coreclr/master/jitstress/arm_cross_checked_ubuntu_jitstress2_jitstressregs0x80_tst/bin/tests/Linux.arm.Checked/JIT/jit64/opt/cse/HugeField1/HugeField1.sh: line 251: 10693 Aborted                 $_DebuggerFullPath "$CORE_ROOT/corerun" $ExePath $CLRTestExecutionArguments
        /n/nReturn code:      1/nRaw output file:      /ssd/j/workspace/dotnet_coreclr/master/jitstress/arm_cross_checked_ubuntu_jitstress2_jitstressregs0x80_tst/bin/tests/Linux.arm.Checked/JIT/jit64/Reports/JIT.jit64/opt/cse/HugeField1/HugeField1.output.txt/nRaw output:/nBEGIN EXECUTION
        /ssd/j/workspace/dotnet_coreclr/master/jitstress/arm_cross_checked_ubuntu_jitstress2_jitstressregs0x80_tst/bin/tests/Linux.arm.Checked/Tests/Core_Root/corerun HugeField1.exe
        Expected: 100
        Actual: 134
        END EXECUTION - FAILED
        Test Harness Exitcode is : 1
        /nTo run the test:/n> set CORE_ROOT=/ssd/j/workspace/dotnet_coreclr/master/jitstress/arm_cross_checked_ubuntu_jitstress2_jitstressregs0x80_tst/bin/tests/Linux.arm.Checked/Tests/Core_Root/n> /ssd/j/workspace/dotnet_coreclr/master/jitstress/arm_cross_checked_ubuntu_jitstress2_jitstressregs0x80_tst/bin/tests/Linux.arm.Checked/JIT/jit64/opt/cse/HugeField1/HugeField1.sh/n
        Expected: True
        Actual:   False
        Stack Trace:
          /mnt/j/workspace/dotnet_coreclr/master/arm_cross_checked_ubuntu/bin/tests/Linux.arm.Checked/TestWrappers/JIT.jit64/JIT.jit64.XUnitWrapper.cs(23909,0): at JIT_jit64._opt_cse_HugeField1_HugeField1_._opt_cse_HugeField1_HugeField1_sh()
      JIT_jit64._opt_cse_HugeField2_HugeField2_._opt_cse_HugeField2_HugeField2_sh [FAIL]
        
        Assert failure(PID 16512 [0x00004080], Thread: 16512 [0x4080]): SUCCEEDED(res)
            File: /mnt/j/workspace/dotnet_coreclr/master/arm_cross_checked_ubuntu/src/unwinder/arm/unwinder_arm.cpp Line: 1540
            Image: /ssd/j/workspace/dotnet_coreclr/master/jitstress/arm_cross_checked_ubuntu_jitstress2_jitstressregs0x80_tst/bin/tests/Linux.arm.Checked/Tests/Core_Root/corerun
        
        /ssd/j/workspace/dotnet_coreclr/master/jitstress/arm_cross_checked_ubuntu_jitstress2_jitstressregs0x80_tst/bin/tests/Linux.arm.Checked/JIT/jit64/opt/cse/HugeField2/HugeField2.sh: line 251: 16512 Aborted                 $_DebuggerFullPath "$CORE_ROOT/corerun" $ExePath $CLRTestExecutionArguments
        /n/nReturn code:      1/nRaw output file:      /ssd/j/workspace/dotnet_coreclr/master/jitstress/arm_cross_checked_ubuntu_jitstress2_jitstressregs0x80_tst/bin/tests/Linux.arm.Checked/JIT/jit64/Reports/JIT.jit64/opt/cse/HugeField2/HugeField2.output.txt/nRaw output:/nBEGIN EXECUTION
        /ssd/j/workspace/dotnet_coreclr/master/jitstress/arm_cross_checked_ubuntu_jitstress2_jitstressregs0x80_tst/bin/tests/Linux.arm.Checked/Tests/Core_Root/corerun HugeField2.exe
        Expected: 100
        Actual: 134
        END EXECUTION - FAILED
        Test Harness Exitcode is : 1
        /nTo run the test:/n> set CORE_ROOT=/ssd/j/workspace/dotnet_coreclr/master/jitstress/arm_cross_checked_ubuntu_jitstress2_jitstressregs0x80_tst/bin/tests/Linux.arm.Checked/Tests/Core_Root/n> /ssd/j/workspace/dotnet_coreclr/master/jitstress/arm_cross_checked_ubuntu_jitstress2_jitstressregs0x80_tst/bin/tests/Linux.arm.Checked/JIT/jit64/opt/cse/HugeField2/HugeField2.sh/n
        Expected: True
        Actual:   False
        Stack Trace:
          /mnt/j/workspace/dotnet_coreclr/master/arm_cross_checked_ubuntu/bin/tests/Linux.arm.Checked/TestWrappers/JIT.jit64/JIT.jit64.XUnitWrapper.cs(23979,0): at JIT_jit64._opt_cse_HugeField2_HugeField2_._opt_cse_HugeField2_HugeField2_sh()
```

**Comment by BruceForstall** [comment]
Same as https://github.com/dotnet/coreclr/issues/22260?

@AndyAyersMS @echesakovMSFT

**Comment by echesakovMSFT** [comment]
I saw only *JIT_jit64.opt_cse_HugeField1_HugeField1._opt_cse_HugeField1_HugeField1_sh* in baseline testing. The second one probably repros only under JitStress=1

**Comment by AndyAyersMS** [comment]
Will investigate along with #22260.

**Comment by AndyAyersMS** [comment]
Fixed by #22381.
